ci-operator/config/openshift/release/openshift-release-master__ci-4.9-upgrade-from-stable-4.8.yaml

```diff
@@ -107,6 +107,20 @@ tests:
       TEST_TYPE: upgrade
       TEST_UPGRADE_OPTIONS: abort-at=99
     workflow: openshift-upgrade-aws
+- as: e2e-aws-upgrade-workload
+  interval: 24h
+  steps:
+    cluster_profile: aws-2
+    env:
+      BASE_DOMAIN: aws-2.ci.openshift.org
+      EXTERNAL_MANIFESTS_POST_INSTALL: "true"
+      EXTERNAL_MANIFESTS_SHA256_HASH: 6007d0022343793a2c40cc4862337eaf9d5e0f4bd8375578b762ee2a5eb298be
+      EXTERNAL_MANIFESTS_URL: https://gist.githubusercontent.com/vrutkovs/1199226040a1942fdbcbe5fa76ab4467/raw/bfc5e67eef6df45f91af76df2a211fb839ebeda9/sample_workload.yaml
```

Comment:
pulling from a separate repo is kind of interesting.  But ideally the monitoring folks will maintain the "how to install user-workload monitoring for CI" manifests going forward, so probably worth putting this somewhere they have access.  Which might work out to be a subdirectory somewhere in this release repo?  But we can sort that out in follow-up work.

Reply:
Oops, looking at [the logs][1] and [your gist][2], I see that the `workload` slug in the job name is "some random user workload, in this case PostgreSQL", and not "user-workload monitoring".  So ignore that previous comment.  I really wish we had a knob for descriptions on jobs...

[1]: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/20116/rehearse-20116-periodic-ci-openshift-release-master-ci-4.9-upgrade-from-stable-4.8-e2e-aws-upgrade-workload/1419947079031066624/artifacts/e2e-aws-upgrade-workload/openshift-manifests-external/build-log.txt
[2]: https://gist.githubusercontent.com/vrutkovs/1199226040a1942fdbcbe5fa76ab4467/raw/bfc5e67eef6df45f91af76df2a211fb839ebeda9/sample_workload.yaml

Reply:
>Which might work out to be a subdirectory somewhere in this release repo?

That would work for me

Reply:
Note for @smarterclayton, this PostgreSQL workload is just a placeholder until we have a real consumer like [a pod spewer][1] to plug in instead.  We can shuffle around the job name as needed to reflect the content being injected in each consuming job.

[1]: https://issues.redhat.com/browse/GRPA-3893